dev-utils/visualise-ui-hierarchy/README.md

```diff
@@ -0,0 +1,59 @@
+# Visualise (the) UI hierarchy (of polyPod features)
+
+## Requirements
+
+Aside from the stuff you probably already have (Node, Make, Python and Git), you
+will need [Graphviz](https://graphviz.org) and [Yarn
+1.x](https://classic.yarnpkg.com/).
+
+## Usage
+
+fhd's fork of the react-component-hierarchy tool generates a, seemingly correct,
```

Comment:
super-extra-premium nit: either you move the comma in front of "correct", or eliminate it, or eliminate them and put "seemingly" in parentheses.

Reply:
Done, similarly at least :D